### ISSUE TITLE
Build does not re-create java/lib directory

### ISSUE THREAD

**Issue by mouse07410** [CardContact/opensc-java#5]
`ant clean` removes `java/lib` directory. Build apparently requires it, but it is not re-created. So the sequence

```
$ ant -f build-without-ivy.xml clean
Buildfile: /Users/uri/src/opensc-java-andreas/build-without-ivy.xml

clean:
   [delete] Deleting directory /Users/uri/src/opensc-java-andreas/build
   [delete] Deleting directory /Users/uri/src/opensc-java-andreas/java/lib

BUILD SUCCESSFUL
Total time: 0 seconds
$ ant -f build-without-ivy.xml 
Buildfile: /Users/uri/src/opensc-java-andreas/build-without-ivy.xml

compile:
    [mkdir] Created dir: /Users/uri/src/opensc-java-andreas/build/bin
    [javac] /Users/uri/src/opensc-java-andreas/build-without-ivy.xml:30: warning: 'includeantruntime' was not set, defaulting to build.sysclasspath=last; set to false for repeatable builds
    [javac] Compiling 38 source files to /Users/uri/src/opensc-java-andreas/build/bin

BUILD FAILED
/Users/uri/src/opensc-java-andreas/build-without-ivy.xml:30: /Users/uri/src/opensc-java-andreas/java/lib does not exist.

Total time: 0 seconds
$
```

fails.

The solution should be amending the `build-without-ivy.xml` to add creation of the missing directory.

**Comment by CardContact** [comment]
Fixed in d97b9fa

**Comment by mouse07410** [comment]
@CardContact sorry, that did not fix the problem. This did:

```
diff --git a/build-without-ivy.xml b/build-without-ivy.xml
index 7f6a368..46d23d6 100644
--- a/build-without-ivy.xml
+++ b/build-without-ivy.xml
@@ -27,6 +27,7 @@

        <target name="compile">
                <mkdir dir="${build}/bin"/>
+               <mkdir dir="${basedir}/java/lib"/>
                <javac encoding="8859_1" srcdir="${basedir}/java/src" destdir="${build}/bin">
                        <classpath refid="project.class.path"/>
                        <exclude name="**/.git"/>
```
